pandas/core/indexes/datetimes.py

```diff
@@ -1991,15 +1992,17 @@ def tz_localize(self, tz, ambiguous='raise', errors='raise'):
         DatetimeIndex(['2018-03-01', '2018-03-02', '2018-03-03'],
         dtype='datetime64[ns]', freq='D')
 
-        #localize DatetimeIndex in US?Eastern time zone.
+        localize DatetimeIndex in US/Eastern time zone.
+
         >>> tz_aware = dti.tz_localize(tz='US/Eastern')
 
         >>> tz_aware
         DatetimeIndex(['2018-03-01 00:00:00-05:00',
         '2018-03-02 00:00:00-05:00', '2018-03-03 00:00:00-05:00'],
         dtype='datetime64[ns, US/Eastern]', freq='D')
 
-        #localize aware time zone into naive time zone.
+        localize aware time zone into naive time zone.
+
         >>> tz_naive = tz_aware.tz_localize(None)
 
         >>> tz_naive
```

Comment:
As you're not doing anything with `tz_naive` more than displaying it, I wouldn't save it to a variable first, I'd show the output direclty.